### PR TITLE
Created budgets donut chart

### PR DESCRIPTION
**Pull request by ewelina-slepko**

### CODE REVIEW

File: src/app/home/panel/cards/dashboard/budgets-statistics/budgets-statistics.component.ts [hunk 1]
```diff
@@ -14,44 +15,140 @@ export class BudgetsStatisticsComponent implements OnInit {
...
+    this.budgetsPercentList = Object.keys(currentMonthBudgetsList)
+      .map((budgetId: string) => (
+        {
+          name: this.budgetsList.filter(budget => budget.id === budgetId).map(({name}) => name).toString(),
+          percentage: currentMonthBudgetsList[budgetId] / budgetsSum,
+          budgetId: budgetId,
+        }
+      ))
+      .sort((a, b) => b.percentage - a.percentage)
+      .map(({...rest}, i) => (
```
Comment (FilipTalaga) [inline src/app/home/panel/cards/dashboard/budgets-statistics/budgets-statistics.component.ts:56]: Should be fine like this:
```ts
.map((rest, i) => (
```

File: src/app/home/panel/cards/dashboard/dashboard.component.scss [hunk 2]
```diff
@@ -8,7 +8,7 @@
 }
 
 .container {
-  margin: 1.5rem;
+  padding: 1.5rem 1.5rem 6rem 1.5rem;
```
Comment (FilipTalaga) [inline src/app/home/panel/cards/dashboard/dashboard.component.scss:11]: You can shorten to:
```ts
  padding: 1.5rem 1.5rem 6rem;
```

File: src/app/shared/utilities.ts [hunk 3]
```diff
@@ -26,17 +25,32 @@ export function transformToDate(timestampValue) {
...
+
       }
     ));
 };
 
+Array.prototype.sumDuplicatedBudgetsAmounts = function() {
+  return [...new Set(this
+    .map(transaction => transaction.budgetId))]
+    .reduce((prev: object, budgetId: string) => {
+      return {
```
Comment (FilipTalaga) [inline src/app/shared/utilities.ts:47]: No point of keeping that `return` here.

**Comment by ewelina-slepko** [comment]
Done :)

**Pull request merged**